### PR TITLE
Table tags and inline style attributes can be used in ckEditor for email templates

### PR DESCRIPTION
**Pull request by boris-brtan**
| Q             | A
| ------------- | ---
|Description, reason for the PR| in administration section for email templates there are ckeditor plugins without ability to include html table tags. This could have additional value for design of the emails and it can have a very good impact on customers. Also we want to have customized mails for demoshop as part of the build.
|New feature| No 
|BC breaks| No 
|Fixes issues| -
|Standards and tests pass| Yes
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

**Review by TomasLudvik — APPROVED** [review]

**Comment by boris-brtan** [comment]
hello, from the test process there was need for allowing inline styles attributes so style tag was removed and all inline css styles attributes were allowed for all tags. Is this OK from CR view @TomasLudvik ?

**Review by TomasLudvik — APPROVED** [review]

File: src/Shopsys/ShopBundle/DataFixtures/Demo/MailTemplateDataFixture.php [hunk 1]
```diff
@@ -0,0 +1,378 @@
...
+use Shopsys\FrameworkBundle\Model\Mail\MailTemplateData;
+use Shopsys\FrameworkBundle\Model\Mail\MailTemplateDataFactoryInterface;
+use Shopsys\FrameworkBundle\Model\Mail\MailTemplateFactoryInterface;
+
+class MailTemplateDataFixture extends AbstractReferenceFixture
+{
+    /**
+     * @var \Shopsys\FrameworkBundle\Component\Domain\Domain
+     */
+    private $domain;
```
Comment (vitek-rostislav) [inline src/Shopsys/ShopBundle/DataFixtures/Demo/MailTemplateDataFixture.php]: Why is the domain `private` and the other properties are `protected`?

**Review by vitek-rostislav — COMMENTED** [review]
Hi @boris-brtan, it seems ok, I tested it as well and the new templates look nice :+1: 
I have only one tiny question, so please check it out. I also noticed that you are missing annotation for some methods but there is no such rule on demoshop so it is probably ok, moreover, after upgrading to a new version of the framework, it will be possible to fix these automatically. Also, the upgrading might be not quite easy after https://github.com/shopsys/shopsys/pull/568 is merged into framework but this is not for us to worry about now.

**Comment by boris-brtan** [comment]
for shops that we UPGRADE we need t olaunch just `php bin/console shopsys:fixtures:load --fixtures=./src/Shopsys/ShopBundle/DataFixtures/Demo --fixtures=./src/Shopsys/ShopBundle/DataFixtures/DemoMultidomain --append --no-interaction`

**Pull request merged**